website/src/content/main-concepts/api-gateway.mdx

```diff
@@ -10,9 +10,52 @@ To solve these issues, the Merchant Center provides an **API Gateway**, which pr
 
 ![HTTP API](/images/mc-http-api.png)
 
-All Merchant Center applications should send requests through the API Gateway, which is available at the same domain of the Merchant Center but with the `mc` subdomain being `mc-api`.
+<Info>
 
-Official domains:
+It is recommended that Custom Applications use the API Gateway to access the commercetools APIs, for security reasons. See [Proxy Endpoints](/main-concepts/proxy-endpoints).
 
-- `https://mc-api.commercetools.com` for projects in **Europe** running on Google Cloud Platform
-- `https://mc-api.commercetools.co` for projects in the **United States** running on Google Cloud Platform
+</Info>
+
+# Cloud regions
+
+The commercetools platform is available in [multiple cloud regions](https://docs.commercetools.com/http-api.html#regions).
+These regions are completely isolated from each other and no data is transferred between them.
+
+<Info>
+
+Platform accounts created for one region are not valid for the other regions. A [signup](https://docs.commercetools.com/getting-started) is required for each region individually.
+In case you need advice in which region your project should be located, please contact [commercetools support](https://support.commercetools.com).
+
+</Info>
+
+## Hostnames
+
+The Merchant Center and the Merchant Center API Gateway are available at the same cloud regions where the commercetools platform runs.
+
+All hostnames are subdomains of `commercetools.com` and follow a specific naming format, including the cloud provider, the cloud region, and the Merchant Center service name.
+
+```
+https://{mcService}.{region}.{cloudProvider}.commercetools.com
+```
+
+- `mcService`: for the Merchant Center front-end the value is `mc`, for the Merchant Center API Gateway the value is `mc-api`.
+- `region`: the region of the cloud provider, see table below.
+- `cloudProvider`: the cloud provider (`gcp`, `aws`, etc.).
+
+| Cloud region | Merchant Center API Gateway hostname |
+| --- | --- |
+| North America (Google Cloud, Iowa) | `mc-api.us-central1.gcp.commercetools.com` |
+| North America (AWS, Ohio) | `mc-api.us-east-2.aws.commercetools.com` |
+| Europe (Google Cloud, Belgium) | `mc-api.europe-west1.gcp.commercetools.com` |
+| Europe (AWS, Frankfurt) | `mc-api.eu-central-1.aws.commercetools.com` |
+
+### Legacy hostnames
+
+Any hostname that does not follow the new naming format is considered a **legacy hostname**.
```

Comment:
Thanks for the compact proposal!  When speaking to  @adinakleine  I realized that we have to keep the information what legacy hostname pointed to what region because it's not actually transparent whether to map ex-EU to GCP or AWS.  So here's a proposal (I'll propose the same change to the HTTP API docs)

```suggestion
Any hostname that does not follow the new naming format is considered a **legacy hostname** but remains available as an alias. `mc{-api}.commercetools.com` is addressing the Google Cloud Belgium Region,  `mc{-api}.commercetools.co` the Google Cloud Iowa Region. 
```

Reply:
This is a bit tricky. We would probably not alias it because of the cookie issue, at least until we figure it out. So I would leave it out for now.

Reply:
@emmenko  you feedback throws both aspects of this change into a bucket.   You can decide not alias like we do in the API hosts, but you still need to tell people what new "Region" name "mc.commercetools.com" originally was and the same for .co.

Reply:
Ok, I thought you didn't want to mention that. Will open a new PR